File: jan21/jan21.java
public class AnimalLegsCounter {
    public static int countLegs(int chickens, int cows, int pigs) {
        int chickenLegs = chickens * 2;
        int cowLegs = cows * 4;
        int pigLegs = pigs * 4;

        return chickenLegs + cowLegs + pigLegs;
    }

    public static void main(String[] args) {
        // Example usage
        int totalLegs = countLegs(2, 3, 5);
        System.out.println("Total number of legs: " + totalLegs);
    }
}

File: jan21/jan21.py
def totalLegs(chickens, cows, pigs):
  chickens *= 2
  cows *= 4
  pigs *= 4
  return chickens + cows + pigs

pigs = int(input("How many pigs are present: "))
cows = int(input("How many cows are present: "))
chickens = int(input("How many chickens are present: "))
print(totalLegs(chickens, cows, pigs))
  
